package.json

```diff
@@ -45,8 +45,5 @@
     "beachball": "^2.31.0",
     "cross-env": "^7.0.3",
     "patch-package": "^6.4.7"
-  },
```

Comment:
I'm not sure why we though it had to be a peerDependency in order to apply the `node_modules` patch. Perhaps that was for a different patch that has since been removed? @rajsite do you remember? Any testing you can think to do besides local build and PR build?

Reply:
The reason is that patch-package is not monorepo aware: https://github.com/ds300/patch-package/issues/356#issuecomment-1334750995

So we are making peer instead of dep to force a single copy* and making it peer at the root so it's pulled to the top.

\* technically this approach still isn't perfect. If the peer at root and a child devDep don't align a second copy will just be made in the child node_modules directory. We wouldn't want it to be a peerDependency in the child because then it becomes a production dependency. We would need a `peerDevDependencies` type concept to avoid that.

Reply:
Some other options:
- Hope and pray - The npm resolution algorithm will generally try to flatten things to the root when possible. Remove the peerDeps and it should "generally" work.
- Try and validate.
  - Maybe scan the lock file and assert there is only one version of the library ever resolved and it's in the root.
  - Maybe make a script that can be run in each package that runs `require.resolve('package-name')` and verifies that the package it resolves to is in the root node_modules folder and not somewhere else.

Reply:
Hope and pray may be fine. At least for these storybook packages I don't think it's likely we introduce multiple copies (ie add stroybook to the angular app or add a package where we develop a storybook plugin)

Reply:
The current lockfile looks good, so hope and pray would be technically fine so I'll mark approved if that seems like the right direction to go

Reply:
I'm going to go ahead with hoping it doesn't break on subsequent installs / signing ourselves up to validate when we do change update storybook. This storybook patches fail quite obviously when they aren't applied (compile error) and one of them already requires revisiting on every upgrade (it applies to a file with a uniquified name that changes in every version). So we're already stuck revalidating it the next time we update. And hopefully the next update allows us to delete them anyway!